spikingjelly/activation_based/ann2snn/delay.py

```
@@ -0,0 +1,256 @@
+                _reset_snn(model)
+                model(x.to(device, non_blocking=True))
+        delay = 0.0
+        for values in ratios.values():
+            if values:
+                delay += sum(values) / len(values)
+    finally:
+        for handle in handles:
+            handle.remove()
+        _reset_snn(model)
+
+    delay_start = int(math.ceil(delay))
```

Ruff (0.15.18)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)